dds_cli/utils.py

```diff
@@ -99,3 +101,16 @@ def format_api_response(response, key, magnitude=None, iec_standard=False):
     else:
         # Since table.add.row() expects a string, try to return whatever is not yet a string but also not numeric as string
         return str(response)
+
+
+def email_validator(ctx, param, value):
+    try:
+        while value is not None:
+            if not re.match(r"(^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$)", value):
+                raise ValueError(value)
+            else:
+                return value
+    except ValueError as err:
+        click.echo("Incorrect email address given: {}".format(err))
+        value = click.prompt("Please try again")
+        return email_validator(ctx, param, value)
```

Comment:
Are we sure there's not a package for this that we can use?

Reply:
Who is we? And is there any specific reason why this solution is wrong or unacceptable?

Reply:
It's not wrong or unacceptable. It's nicely written. But the general idea is to "not reinvent the wheel" so if there's another package (maybe it's possible to use the `marshmallow.validator.Email` but I'm not sure if it's usable here) or function in a already used package that does this exact thing (checks if an email has the correct format) then we should use that and not write our own function.

Reply:
Yes. The solution is sufficient and no it's not rocket science. All I wanted to know was if there was another shorter way of doing this, and since there apparently isn't then using this is fine.